ENGAGEHF/HeartHealth/Shared/ListSection/MeasurementListHeader.swift

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct MeasurementListHeader: View {
+    let addingEnabled: Bool
+    let measurementType: GraphSelection
+    @Binding var addingMeasurement: GraphSelection?
+    
+    
+    var body: some View {
+        HStack {
+            Text("All Data")
+                .font(.title3.bold())
+            Spacer()
+            if addingEnabled {
+                TriggerNewMeasurementButton(measurementType: measurementType, addingMeasurement: $addingMeasurement)
```

Comment:
I am wondering if we can just add the sheet in here and even add the adding enabled calculation just based on the measurementType.

So we could make the `@State` private here and reduce the logic of having to pass it up and down the view hierarchy.